src/middlewares/role.middleware.js

```diff
@@ -21,6 +22,20 @@ const roles = {
   checkRequester(req, res, next) {
     if (req.userData.role === 'requester') return next();
     return notAuthorized(res);
+  },
+  checkAdmin(req, res, next) {
+    const token = helper.getToken(req);
+    const data = helper.verifyToken(token);
+
+    if (data.error) {
```

Comment:
great job @urbanishimwe  but return the real data from the error so that a client can know which kind of message.

Reply:
I just copy and paste the code which was already there